packages/react/src/components/form/Field/FieldAffix.tsx

```diff
@@ -2,24 +2,27 @@ import cl from 'clsx/lite';
 import type { HTMLAttributes } from 'react';
 import { forwardRef } from 'react';
 
-export type FieldAffixWrapperProps = Omit<
-  HTMLAttributes<HTMLDivElement>,
-  'prefix'
->;
+export type FieldAffixesProps = Omit<HTMLAttributes<HTMLDivElement>, 'prefix'>;
 
-export const FieldAffixWrapper = forwardRef<
-  HTMLDivElement,
-  FieldAffixWrapperProps
->(function FieldAffixWrapper({ className, ...rest }, ref) {
-  return (
-    <div className={cl('ds-field-affix', className)} ref={ref} {...rest} />
-  );
-});
+export const FieldAffixes = forwardRef<HTMLDivElement, FieldAffixesProps>(
+  function FieldAffixes({ className, ...rest }, ref) {
+    return (
+      <div className={cl('ds-field-affixes', className)} ref={ref} {...rest} />
+    );
+  },
+);
 
 export type FieldAffixProps = Omit<HTMLAttributes<HTMLDivElement>, 'prefix'>;
 
 export const FieldAffix = forwardRef<HTMLSpanElement, FieldAffixProps>(
-  function FieldAffix(rest, ref) {
-    return <span aria-hidden='true' ref={ref} {...rest} />;
+  function FieldAffix({ className, ...rest }, ref) {
+    return (
+      <span
+        className={cl('ds-field-affix', className)}
+        aria-hidden='true'
```

Comment:
Should we put this after `rest` so we make sure the user can't turn this off?

Reply:
mm, I think if they really want to set `aria-hidden="false"` they are on their own and we should not prevent it being possible ☺️

Reply:
fair enough, makes sense to me